ush/config_defaults.sh

```diff
@@ -426,13 +426,21 @@ WFLOW_LAUNCH_LOG_FN="log.launch_FV3LAM_wflow"
 # Set up frequenency or list of the forecast hours that FV3 should
 # generate the restart files.
 #
+# POSTPROC_LEN_HRS:
+# The length of post process, in integer hours.
+#
+# POSTPROC_LONG_LEN_HRS:
+# The length of long post process, in integer hours.
+#
 #-----------------------------------------------------------------------
 #
 DATE_FIRST_CYCL="YYYYMMDD"
 DATE_LAST_CYCL="YYYYMMDD"
 CYCL_HRS=( "HH1" "HH2" )
-BOUNDARY_LEN_HRS="24"
-BOUNDARY_LONG_LEN_HRS="24"
+BOUNDARY_LEN_HRS="1"
+BOUNDARY_LONG_LEN_HRS="1"
+POSTPROC_LEN_HRS="1"
```

Comment:
Should postproc_len_hrs takes a default value of 0? i.e, only post-process the f00 file which is so under the RTMA situation.

Reply:
If default value does not cause any trouble, we can use 0.